src/main/java/org/programmers/springorder/customer/repository/FileCustomerRepository.java

```diff
@@ -0,0 +1,67 @@
+package org.programmers.springorder.customer.repository;
+
+import org.programmers.springorder.console.Console;
+import org.programmers.springorder.consts.ErrorMessage;
+import org.programmers.springorder.customer.model.Customer;
+import org.programmers.springorder.customer.model.CustomerType;
+import org.programmers.springorder.voucher.model.Voucher;
+import org.programmers.springorder.voucher.model.VoucherType;
+import org.programmers.springorder.voucher.repository.FileVoucherRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Repository;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Repository
+public class FileCustomerRepository implements CustomerRepository{
+
+    @Value(("${customerListFilePath}"))
+    private String filePath;
+    private final Logger logger = LoggerFactory.getLogger(FileVoucherRepository.class);
```

Comment:
```suggestion
    private static final Logger logger = LoggerFactory.getLogger(FileCustomerRepository.class);
```

1. 상수를 명확히 선언해주시면 좋을거 같아요.
2. 변수보다 상수를 위에 두는게 좋지 않을까요?